### ISSUE TITLE
Show symbol while terminal is working

### ISSUE THREAD

**Issue by snowparrot** [elementary/terminal#317]
For sucessfull or none-sucessfull finishing of a command Terminal shows a symbol in the tabs. It is convenient if there would be a symbol for working command. 

I propose a symbolic hourglass.

**Comment by cassidyjames** [comment]
I don't think we can know if something is actively "working" versus waiting for user input, but I'm not sure. If so, something like a loading spinner could make sense, similar to a web browser.

**Comment by jeremypw** [comment]
Re-opening as the animated spinner proved distracting. May use a static icon at some stage.
